packages/dds/tree/src/shared-tree/treeApiAlpha.ts

```diff
@@ -206,9 +208,12 @@ export interface TreeAlpha {
 }
 
 /**
+ * Extensions to {@link Tree} and {@link TreeBeta} which are not yet stable.
+ * @remarks
+ * See {@link (TreeAlpha:interface)}.
```

Comment:
Nit: you could also do the following instead, for simplicity
```suggestion
 * @see {@link (TreeAlpha:interface)}.
```